### PR TITLE
Badges started

### PR DESCRIPTION
**Pull request by dreamwasp**
### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [ABC-123]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.e6af09.0<br/>@codecademy/gamut@57.5.2-alpha.e6af09.0<br/>@codecademy/gamut-icons@9.34.1-alpha.e6af09.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.e6af09.0<br/>@codecademy/gamut-kit@0.6.455-alpha.e6af09.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.e6af09.0<br/>@codecademy/gamut-styles@17.1.2-alpha.e6af09.0<br/>@codecademy/gamut-tests@5.1.4-alpha.e6af09.0<br/>@codecademy/macros@3.0.5-alpha.e6af09.0<br/>@codecademy/styleguide@68.1.4-alpha.e6af09.0<br/>@codecademy/variance@0.21.5-alpha.e6af09.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.141d14.0<br/>@codecademy/gamut@57.5.2-alpha.141d14.0<br/>@codecademy/gamut-icons@9.34.1-alpha.141d14.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.141d14.0<br/>@codecademy/gamut-kit@0.6.455-alpha.141d14.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.141d14.0<br/>@codecademy/gamut-styles@17.1.2-alpha.141d14.0<br/>@codecademy/gamut-tests@5.1.4-alpha.141d14.0<br/>@codecademy/macros@3.0.5-alpha.141d14.0<br/>@codecademy/styleguide@68.1.4-alpha.141d14.0<br/>@codecademy/variance@0.21.5-alpha.141d14.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://67473ed3b738ce178124da71--gamut-preview.netlify.app>https://67473ed3b738ce178124da71--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/67473ed3b738ce178124da71>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6747412c041fd02a429514e3--gamut-preview.netlify.app>https://6747412c041fd02a429514e3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6747412c041fd02a429514e3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.322d51.0<br/>@codecademy/gamut@57.5.2-alpha.322d51.0<br/>@codecademy/gamut-icons@9.34.1-alpha.322d51.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.322d51.0<br/>@codecademy/gamut-kit@0.6.455-alpha.322d51.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.322d51.0<br/>@codecademy/gamut-styles@17.1.2-alpha.322d51.0<br/>@codecademy/gamut-tests@5.1.4-alpha.322d51.0<br/>@codecademy/macros@3.0.5-alpha.322d51.0<br/>@codecademy/styleguide@68.1.4-alpha.322d51.0<br/>@codecademy/variance@0.21.5-alpha.322d51.0<br/></pre>

File: packages/styleguide/.storybook/preview.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { withEmotion } from './theming/GamutThemeProvider';
 import { breakpoints } from '@codecademy/gamut-styles';
 import { DocsContainer } from './components/Elements/DocsContainer';
 
+// start a small amount of work on atoms
```
Comment (aresnik11) [inline packages/styleguide/.storybook/preview.ts:8]: delete?

File: packages/styleguide/src/lib/Atoms/Badge/Badge.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+<Canvas of={BadgeStories.TertiaryFill} />
+
+### Accent
+
+Use the accent variant to draw attention to important details or statuses with a noticeable but less dominant emphasis than the primary badge.
+
+<Callout text="The accent variant does not respond to Color Mode." />
+
+<Canvas of={BadgeStories.Accent} />
+
```
Comment (aresnik11) [inline packages/styleguide/src/lib/Atoms/Badge/Badge.mdx:94]: do we also want to add stories for size and leading icon? thats what we have on prod. i can do it on my branch

Comment (dreamwasp) [reply packages/styleguide/src/lib/Atoms/Badge/Badge.mdx:94]: yeah 100%, i was just mindlessly going off of the figma lol

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.2.1-alpha.850380.0<br/>@codecademy/gamut@57.5.2-alpha.850380.0<br/>@codecademy/gamut-icons@9.34.1-alpha.850380.0<br/>@codecademy/gamut-illustrations@0.49.2-alpha.850380.0<br/>@codecademy/gamut-kit@0.6.455-alpha.850380.0<br/>@codecademy/gamut-patterns@0.9.17-alpha.850380.0<br/>@codecademy/gamut-styles@17.1.2-alpha.850380.0<br/>@codecademy/gamut-tests@5.1.4-alpha.850380.0<br/>@codecademy/macros@3.0.5-alpha.850380.0<br/>@codecademy/styleguide@68.1.4-alpha.850380.0<br/>@codecademy/variance@0.21.5-alpha.850380.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://674f25b302f2b32170845af3--gamut-preview.netlify.app>https://674f25b302f2b32170845af3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/674f25b302f2b32170845af3>Deploy Logs</a></p>

**Review by aresnik11 — APPROVED** [review]
Good to merge in, I will add those other badge stories

**Pull request merged**